src/styles/article-reader.scss

```
@@ -51,6 +51,10 @@
     margin-bottom: var(--space-08);
   }
 
+  h2 {
+    scroll-margin-top: var(--space-128);
+  }
+
   h3 {
     margin-bottom: var(--space-24);
```

We also have h3 headers in the TableOfContents, like [here](https://staging.nodejs.dev/1676/learn/understanding-javascript-promises#cascading-errors). We should add this property here too I think.

Otherwise, LGTM

```suggestion
    margin-bottom: var(--space-24);
    scroll-margin-top: var(--space-128);
```

---

Nice catch, thank you!